extensions/canvasMedical/v1/actions/createClaim/config/fields.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  DateOnlySchema,
+  FieldType,
+  type Field,
+} from '@awell-health/extensions-core'
+import { z, type ZodTypeAny } from 'zod'
+
+export const fields = {
```

Comment:
I would recommend adding a comment linking to the source documentation for these fields